docs/modules/user/pages/quickstart.adoc

```diff
@@ -501,6 +501,172 @@ And if you have a multi GPU setup, don't forget to set the below env variable
+
+This example is based on Nvidia (Manual), you will need to create the Nvidia volume first before starting the service:
+
+1.{empty} Create a new image that will contain the Nvidia driver files:
+
+[source,bash]
+....
+sudo curl https://raw.githubusercontent.com/games-on-whales/gow/master/images/nvidia-driver/Dockerfile | sudo podman build -t gow/nvidia-driver:latest -f - --build-arg NV_VERSION=$(cat /sys/module/nvidia/version) .
+....
+
+2.{empty} Mount the image and create the new volume:
+
+[source,bash]
+....
+sudo podman create --rm --mount source=nvidia-driver-vol,destination=/usr/nvidia gow/nvidia-driver:latest sh
+....
+
+Check volume exists with:
+
+[source,bash]
+....
+sudo docker volume ls | grep nvidia-driver
+
+local     nvidia-driver-vol
+....
+
+3.{empty} This is an additional step for Podman. Start the temporary container you just created. `sudo podman ps -a` will show a contianer using a temporary name (e.g. `musing_pascal`) and your local `gow/nvidia-driver` image. Only once this container is started will your new volume be populated. Run:
+
+[source,bash]
+....
+sudo podman start musing_pascal
+....
+
+You can verify the contents of of your `nvidia-driver-vol` by running `sudo podman volume inspect nvidia-driver-vol` and then browsing the `Mountpoint`: `sudo la -la /var/lib/containers/storage/volumes/nvidia-driver-vol/_data`. You should see a non empty folder:
+
+[source,bash]
+....
+total 12
+drwxr-xr-x. 6 root root   54 Oct 27 22:06 .
+drwx------. 3 root root   19 Oct 28 14:40 ..
+drwxr-xr-x. 2 root root 4096 Oct 27 22:06 bin
+drwxr-xr-x. 5 root root 4096 Oct 28 14:44 lib
+drwxr-xr-x. 4 root root 4096 Oct 27 22:06 lib32
+drwxr-xr-x. 7 root root   77 Oct 27 22:07 share
+....
+
+4.{empty} Use the example below to create a `wolf.container` file in `/etc/containers/systemd`, then run `sudo systemctl daemon-reload` and `sudo systemctl start wolf.service` to start.
+
+[source,bash]
+....
+[Unit]
+Description=Wolf / Games On Whales
+Requires=network-online.target podman.socket
+After=network-online.target podman.socket
+
+[Service]
+TimeoutStartSec=900
+ExecStartPre=-/usr/bin/mkdir -p /tmp/sockets
+ExecStartPre=-/usr/bin/podman rm --force WolfPulseAudio
+Restart=on-failure
+RestartSec=5
+StartLimitBurst=5
+
+[Container]
+AutoUpdate=registry
+ContainerName=%N
+HostName=%N
+Image=ghcr.io/games-on-whales/wolf:stable
+AddCapability=CAP_SYS_PTRACE
+AddCapability=CAP_NET_ADMIN
+Network=host
+SecurityLabelDisable=true
+PodmanArgs=--ipc=host --device-cgroup-rule "c 13:* rmw"
+AddDevice=/dev/dri
+AddDevice=/dev/uinput
+AddDevice=/dev/uhid
+AddDevice=/dev/nvidia-uvm
+AddDevice=/dev/nvidia-uvm-tools
+AddDevice=/dev/nvidia-caps/nvidia-cap1
+AddDevice=/dev/nvidia-caps/nvidia-cap2
+AddDevice=/dev/nvidiactl
+AddDevice=/dev/nvidia0
+AddDevice=/dev/nvidia-modeset
+Volume=nvidia-driver-vol:/usr/nvidia
+Volume=/dev/:/dev/:rw
+Volume=/run/udev:/run/udev:rw
+Volume=/etc/wolf/:/etc/wolf:z
+Volume=/run/podman/podman.sock:/var/run/docker.sock:ro
```

Comment:
We should probably add a line or two explaining that `/run/podman/podman.sock` might be missing, possibly linking to https://github.com/containers/podman/blob/main/docs/tutorials/socket_activation.md

Reply:
Added some additional pointers for starting the socket